pilot/pkg/model/push_context.go

```diff
@@ -495,7 +495,22 @@ func (ps *PushContext) DestinationRule(proxy *Proxy, service *Service) *Config {
 	}
 
 	// FIXME: this code should be removed once the EDS issue is fixed
+	var configPtr *Config
 	if proxy == nil {
```

Comment:
I just search all the code from master, and find that currently proxy can not be nil. Maybe it's time to remove this branch. @rshriram What do you think?